internal/cronsumer.go

```diff
@@ -71,6 +73,7 @@ func (k *kafkaCronsumer) Listen(ctx context.Context) {
 }
 
 func (k *kafkaCronsumer) Stop() {
+	close(k.messageChannel)
```

Comment:
I put this line to stop goroutines that listen to this channel; otherwise, it causes goroutine leaks.

Reply:
Context cancel should be enough for stopping goroutine leak. Isn't it ?

Reply:
No, because we are using this channel very separately. I mean from the point of initializing and assigning arrived messages.